skimage/feature/_hoghistogram.pyx

```diff
@@ -0,0 +1,130 @@
+# cython: cdivision=True
+# cython: boundscheck=False
+# cython: wraparound=False
+
+import numpy as np
+cimport numpy as cnp
+
+cdef float cell_hog(cnp.float64_t[:, :] magnitude,
+        cnp.float64_t[:, :] orientation,
```

Comment:
Please indent these and following to match the opening `(` of the function definition.